### ISSUE TITLE
Missing release v29

### ISSUE THREAD

**Issue by bebeo92** [nextcloud-releases/fulltextsearch_elasticsearch#2]
Hi repo owner

I see that you create a tag for release 29 but I think you forget create an actual release that cause an issue with nextcloud when try to install it
![image](https://github.com/nextcloud-releases/fulltextsearch_elasticsearch/assets/9253836/cc6a69d6-f793-4df3-83b3-3a926214953c)

**Comment by kmorber** [comment]
Same for me.

**Comment by jurgenhaas** [comment]
Same problem here, artefacts are missing.

**Comment by bebeo92** [comment]
@ArtificialOwl @skjnldsv could you guys take a look ?

**Comment by zenhighzer** [comment]
same here :)

**Comment by SinisterCrayon** [comment]
Ditto :)

**Comment by neocult-de** [comment]
+1

**Comment by bebeo92** [comment]
![image](https://github.com/nextcloud-releases/fulltextsearch_elasticsearch/assets/9253836/f5f72801-32cd-4b17-b379-bffbf3bc1d2e)
Seeing new error not sure it is related @ArtificialOwl

**Comment by bebeo92** [comment]
Work for me so I close this issue

**Comment by zenhighzer** [comment]
Since I enabled the "fulltext-stack" I got an error running the 5min-php-cron-job: 

nextcloud (29.0.0) as container
installed apps: 
```
[...]
 - files_fulltextsearch: 28.0.0
 - files_fulltextsearch_tesseract: 27.0.0
 - fulltextsearch: 28.0.1
 - fulltextsearch_elasticsearch: 29.0.0
[...]

```
Error while running 'docker exec -u www-data nextcloud php -f /var/www/html/cron.php': 

```
Error: Class "OC\BackgroundJob\TimedJob" not found in /var/www/html/custom_apps/fulltextsearch/lib/Cron/Index.php:51
Stack trace:
#0 /var/www/html/lib/composer/composer/ClassLoader.php(576): include()
#1 /var/www/html/lib/composer/composer/ClassLoader.php(427): Composer\Autoload\{closure}('/var/www/html/c...')
#2 [internal function]: Composer\Autoload\ClassLoader->loadClass('OCA\\FullTextSea...')
#3 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(126): ReflectionClass->__construct('OCA\\FullTextSea...')
#4 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(146): OC\AppFramework\Utility\SimpleContainer->resolve('OCA\\FullTextSea...')
#5 /var/www/html/lib/private/AppFramework/DependencyInjection/DIContainer.php(470): OC\AppFramework\Utility\SimpleContainer->query('OCA\\FullTextSea...')
#6 /var/www/html/lib/private/ServerContainer.php(155): OC\AppFramework\DependencyInjection\DIContainer->queryNoFallback('OCA\\FullTextSea...')
#7 /var/www/html/lib/private/AppFramework/Utility/SimpleContainer.php(64): OC\ServerContainer->query('OCA\\FullTextSea...')
#8 /var/www/html/lib/public/Server.php(55): OC\AppFramework\Utility\SimpleContainer->get('OCA\\FullTextSea...')
#9 /var/www/html/lib/private/BackgroundJob/JobList.php(326): OCP\Server::get('OCA\\FullTextSea...')
#10 /var/www/html/lib/private/BackgroundJob/JobList.php(235): OC\BackgroundJob\JobList->buildJob(Array)
#11 /var/www/html/cron.php(163): OC\BackgroundJob\JobList->getNext(true)
#12 {main}
```

**Comment by SinisterCrayon** [comment]
> Since I enabled the "fulltext-stack" I got an error running the 5min-php-cron-job:

You need to upgrade your fulltextsearch components to 29.0.0. This is a known issue with running the 28.0 code on Nextcloud 29.0

EDIT: NVM... looks like their "fix" was to revert to installing 28.0.1 on NextCloud 29. So yeah... FTS is broken on NextCloud 29 for now.

https://github.com/nextcloud/backup/issues/663
